### PR TITLE
Tiaas 2.1

### PR DESCRIPTION
**Pull request by neoformit**
Update tiaas role to allow deployment of new tiaas release.
When admins re-install the role, they should automatically install the new tiaas2 release.

Q: I've forgotten - how are we going to manage version/branching for tiaas2 so that admins doesn't get a surprise the next time they run their playbook?

### New variables  & customize HTML
Check out README.md

### CODE REVIEW

**Comment by hexylena** [comment]
A: we discussed branches, and just having everyone coordinate the update since we have most of them in one room at euro galaxy days. 

given that some sites already pin a specific version, and the others I have control over (or know the folks) I think the second is an acceptable option here.

**Comment by hexylena** [comment]
Since the window passed (oh well) let's change the branch to main2 and be done, let's merge :)

File: defaults/main.yml [hunk 1]
```diff
@@ -13,7 +16,7 @@ tiaas_static_dir: "{{ tiaas_dir }}/static"
 tiaas_venv_dir: "{{ tiaas_dir }}/venv"
 tiaas_user: tiaas
 tiaas_group: tiaas
-tiaas_repo: "https://github.com/usegalaxy-eu/tiaas2"
+tiaas_repo: "https://github.com/galaxyproject/tiaas2"
 tiaas_version: main
```
Comment (hexylena) [inline defaults/main.yml]:
```suggestion
tiaas_version: main2
```

**Review by hexylena — APPROVED** [review]

**Comment by neoformit** [comment]
Awesome, I'll update out role on dev and see how this deploys. Thanks @hexylena!

**Comment by neoformit** [comment]
Hey, does this automatically update the Ansible-Galaxy role, or is that manual?
EDIT: think I answered my own question: https://galaxy.ansible.com/galaxyproject/tiaas2

**Comment by neoformit** [comment]
Ok it's deployed on https://dev.usegalaxy.org.au/tiaas/ and looking good :)

**Comment by hexylena** [comment]
Looks absolutely fantastic! You're switched over to gunicorn too? Just want to make sure that wasn't too painful

**Comment by neoformit** [comment]
Hmm no we're still getting the `wsgi` command in `run.sh` at role v2.1.0. Could you make a new release with the `gunicorn` `run.sh` template? Was that committed recently? :thinking: 

image

**Comment by hexylena** [comment]
Oh, you're right, it was after. 2.1.1 is on its way.

**Pull request merged**